tests/streaming/test_streaming_conversation.py

```diff
@@ -253,7 +259,9 @@ async def test_transcriptions_worker_ignores_utterances_before_initial_message(
             is_final=True,
         ),
     )
-    assert await _consume_worker_output(streaming_conversation.transcriptions_worker) is None
+    transcriptions_worker_consumer = QueueConsumer()
```

Comment:
You should remove line 239 right? And attach the consumer earlier? The consumer otherwise doesn't get the `consume_nonblocking()` results above

Reply:
yup, absolutely right!